template/java8/model/src/test/java/RequestTest.java

```diff
@@ -10,14 +13,14 @@
 	@Test
 	public void testSingleRequestParameterGetSet()
 	{
-		Request request = new Request(null,null,"testParam=testParamValue", null);
+		Request request = new Request(null,"testParam=testParamValue", null, null);
```

Comment:
Does this need formatting? ` Request(null,"testParam=testParamValue", null, null);`

Reply:
Yes, I will fix that.